uint8 isLeftWheelOnline
uint8 isRightWheelOnline
uint8 isWheelEnabled
uint8 isForceShutdownTriggered
float32 tiltAngle
float32 zMove